data/i18n/en/en.toml

```
@@ -18,6 +18,9 @@ other = "Show More Posts..."
 [banner_acknowledgement]
 other = "Hide this notice"
 
+[case_study_prefix]
+other = "Case study: "
```

The extra spacing in the `case_study_prefix` value is reflecting as unnecessary space in the rendered case study page(s).  Perhaps we could remove it for a cleaner presentation.

Reference Links: [Preview Case Study](https://deploy-preview-45945--kubernetes-io-main-staging.netlify.app/case-studies/babylon/) | [Current Website Case Study](https://kubernetes.io/case-studies/babylon/)

```suggestion
other = "Case study:"
```